src/code.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+set -e -x -o pipefail
+main() {
+# download all inputs
+mark-section "Download all inputs"
+dx-download-all-inputs --parallel
+
+# get project name and folder name
+project_name=$(dx describe $project_id --json | jq '.name')
+folder_name=$(echo $project_name | sed 's/_[^_]*$//g')
+folder_name=$(printf '%s\n' "${folder_name#*_}")
```

Comment:
Hi @natashapinto , I want to highlight about this line in case you start reviewing the code. This line of code is to get the sub-folder name inside the ADX project. For example - in the project "002_240726_NB552085_0328_AHFYLHAFX7_ADX24028", the sub-folder name is "240726_NB552085_0328_AHFYLHAFX7". So, this command line gets the sub-folder name by extracting the string between the first and last underscores as I assume that it is the naming format for all ADX projects and sub-folders. But please do let me know if my assumption is not true for all cases. I may update code to capture the sub-folder name accordingly.